src/main/kotlin/org/rust/ide/refactoring/move/common/RsMoveUtil.kt

```diff
@@ -211,6 +211,25 @@ fun <T : RsElement> movedElementsDeepDescendantsOfType(
         }
 }
 
+/**
+ * Updates `pub(in path)` visibility modifier.
+ * We replace `path` with common parent module of `newParent` and target of old `path`.
+ */
+fun RsVisRestriction.updateScopeIfNecessary(psiFactory: RsPsiFactory, newParent: RsMod) {
```

Comment:
This is very similar to `org.rust.ide.refactoring.move.RsMoveFilesOrDirectoriesProcessorKt#updateScopeIfNecessary` 🤔

Reply:
Thank you, I forgot to delete old version